sudoku_solver.py

```diff
@@ -0,0 +1,88 @@
+class Board:
+    def __init__(self):
+        self.sudoku_matrix = None
+        self.numbers = []
+        self.count = None
+
+
+    def setup(self): # Define a sudoku puzzle here
+        self.sudoku_matrix = [  [ 2, 0, 0, 0, 4, 0, 7, 0, 0, ], 

```

Comment:
The setup function is pointless. You might as well put this stuff in the init function. Also maybe you could accept a grid as an input argument.